cli/src/cmd/forge/test.rs

```diff
@@ -346,6 +348,21 @@ impl TestOutcome {
     }
 }
 
+fn block_summary(duration: Duration, outcome: TestOutcome) {
```

Comment:
can we make this a function of `TestOutcome` instead: `TestOutcome::summary -> String`?

Reply:
Sure—are there any Rust specific rules of thumb on when to do this vs. use free functions? The only slightly tricky thing here is the `duration`, which is not defined for a `TestOutcome` with multiple results.

I left `duration` as an external argument to this function, but we _could_ make the `duration` the sum of all the `SuiteResult` durations for a `TestOutcome`, if it is OK to sum up multiple `Duration`s in that way. What do you think?